desktop/apps/partner2/components/fixed_cells_count_carousel/test/view.coffee

```diff
@@ -119,13 +119,13 @@ describe 'FixedCellsCountCarousel', ->
         articles2 = results: [fabricate('article')]
         Backbone.sync
           .onCall 0
-          .yieldsTo 'success', articles1
           .returns Promise.resolve articles1
+          .yieldsTo 'success', articles1
 
         Backbone.sync
           .onCall 1
-          .yieldsTo 'success', articles2
           .returns Promise.resolve articles2
+          .yieldsTo 'success', articles2
```

Comment:
Just curious but why switch the order in the above? Was it a previous bug?

Reply:
I played around with this for a while and it seems like the order of the chaining mattered because the tests wouldn't pass without this order. Now that I look at this again though, I think we can use `.resolves` on these kinds of tests. Will update!